README.md

```diff
@@ -196,6 +199,7 @@ function CommonRegistry(opts){
 util.inherits(CommonRegistry, DefaultRegistry);
 
 CommonRegistry.prototype.init = function(takerInst){
+  DefaultRegistry.prototype.init.call(this, takerInst);
```

Comment:
The goal is to never change the DefaultRegistry and for it to only provide the bare minimum for people that don't want to overwrite certain methods.  Thanks for thinking about the future though.